3_javascript/javascript1/week3/noteTakingApp/script.js

```diff
@@ -0,0 +1,46 @@
+// Save a note
+const notes = [];
+
+function saveNote(content, id, important) {
+  notes.push({
+    id: id,
+    content: content,
+    important: important
+  })
+}
+saveNote("Pick up groceries", 1 ,true);
+saveNote("Do laundry", 2, false);
+saveNote("Do Homeworks", 3, true);
+console.log(notes); 
+
+// get note
+function getNote(id) {
+    for(const note of notes){
+        if(note.id === id)
+        return note.content;
+    }
+  } 
+const firstNote = getNote(1);
+console.log(firstNote);
+
+// Log out notes
+function logOutNotesFormatted() {
+    let allNotes = ""
+    for(const note of notes){
+        allNotes += `The note with id: ${note.id}, has the following note text: ${note.content}\n`
```

Comment:
`allNotes +=`
is there any reason to use + and = sign the value to variable

Reply:
Hi, I want to show all of the notes, and if I use return instead I will get just the first note. (maybe I can use console.log instead of return.)
what is your suggestion?

Reply:
yes you are right , was my bad . :-)

Reply:
alright, thanks